stage1/init/common/units.go

```diff
@@ -179,6 +171,131 @@ func ImmutableEnv(p *stage1commontypes.Pod, interactive bool, privateUsers strin
 	return w.Error()
 }
 
+// SetupAppIO prepares all properties related to streams (stdin/stdout/stderr) and TTY
+// for an application service unit
+func (w *UnitWriter) SetupAppIO(p *stage1commontypes.Pod, ra *schema.RuntimeApp, binPath string, interactive bool, debug bool, opts ...*unit.UnitOption) []*unit.UnitOption {
```

Comment:
I know this is a WIP but I would love this to have a really good docblock, plus a reference to whatever other docs you write. Being nice to the future programmer who comes and wonders what the heck we're doing.